src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
+import moviesFromServer from './api/movies.json';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
-import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const getFilteredMovies = () => {
+    if (!query) {
+      return moviesFromServer;
+    }
+
+    const isMatch
+    = (text: string,
+      textQuery: string) => text.toLowerCase().includes(textQuery);
+
+    const lowerCaseQuery = query.toLowerCase().trim();
```

Comment:
It would be better to create a function to format the strings instead, as isMatch function doesn't make your code cleaner